docs/arch/adr-006-use-mermaid-for-simple-smart-answer-visualisation.md

```diff
@@ -0,0 +1,35 @@
+# ADR 006 - Use Mermaid for Simple Smart Answer Visualisation
+
+## Status
+Proposed
+
+## Context
+Following user research it was identified that Content Designers (CDs) were struggling to understand the flow of a smart answer, communicate this with colleagues and confidently edit it. 
```

Comment:
Was it mentioned that the CDs have arrived at solutions outside of the system to visualise the more complex flows? And the ask is to bring that into the system to reduce context switching and extra overhead in converting formats. Feels like there's a sentence or two missing between the stated problem and the evaluation as well, what it is we're looking to make a decision on.